app/code/BodenkoVV/AskQuestion/Cron/ChangeStatusOnCron.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace BodenkoVV\AskQuestion\Cron;
+
+use BodenkoVV\AskQuestion\Model\Question;
+use BodenkoVV\AskQuestion\Model\ResourceModel\Question\CollectionFactory;
+use Psr\Log\LoggerInterface as LoggerInterfaceAlias;
+
+class ChangeStatusOnCron
+{
+    /** @var CollectionFactory  */
+    protected $questionsFactory;
+
+    /** @var LoggerInterfaceAlias */
+    private $logger;
+
+    /** @var int  */
+    public $cronDayAlert;
+
+    /**
+     * ChangeStatusOnCron constructor.
+     * @param LoggerInterfaceAlias $logger
+     * @param CollectionFactory $askQuestionsFactory
+     */
+    public function __construct(
+        LoggerInterfaceAlias $logger,
+        CollectionFactory $askQuestionsFactory
+    )
+    {
+        $this->logger = $logger;
+        $this->questionsFactory = $askQuestionsFactory;
+        $this->cronDayAlert = 3;
```

Comment:
Don't hard-code such values, please create the method getDaysNumberBeforeStatusIsChanged or something like this, and you can return 3 there. In the future you can refactor this method to retrieve it from magento config. Or you can get it from config now, we had this topic discussed.

Reply:
+